CHANGELOG.md

```diff
@@ -14,6 +14,12 @@
 - **Amazon SNS**
   - Updated model to support latest Amazon SNS features such as `TagResource` function. This resolves [Issue# 1020](https://github.com/aws-amplify/aws-sdk-android/issues/1020). 
 
+### Bug Fixes
+
+- **Amazon S3**
```

Comment:
Update changelog to refer to the relevant issue

Reply:
It did, but it was incorrectly formatted on a new line. will fix